.ci/cico-happy-path-pr-test.sh

```diff
@@ -102,7 +103,7 @@ else
     getOpenshiftLogs
     archiveArtifacts "che-plugin-registry-prcheck"
     echo "Failed while starting chectl. Please check logs available in http://artifacts.ci.centos.org/devtools/che/che-plugin-registry-prcheck/${ghprbPullId}/"
-    exit 1337
+    exit 133
```

Comment:
This may be a dumb question, but what's the significance of this change?

Reply:
Shellcheck says you can't return a number greater than 255.

So assuming whoever wrote this wanted a value other than 1 I figured I'd change it to a less "leet" value that was kinda "lee".

Reply:
Sounds fair to me!